testing/cli/test_cli_sct_register_multimodal.py

```diff
@@ -148,3 +148,15 @@ def test_sct_register_multimodal_with_labels(capsys, tmp_path, algo):
     # NB: Right now, a warning will be thrown regardless of whether `algo` is explicitly
     #     specified by the user, because `algo` has a default, non-empty setting.
     assert "has no effect for 'type=label' registration." in capsys.readouterr().out
+
+
+def test_sct_register_multimodal_with_qc_without_dseg(capsys, tmp_path):
+    """
+    Test if an error is raised when using QC ('-qc') without providing a destination segmentation ('-dseg').
+    """
+    with pytest.raises(SystemExit) as pytest_wrapped_e:
+        sct_register_multimodal.main(['-i', sct_test_path('t2', 't2.nii.gz'),
+                                      '-d', sct_test_path('t2', 't2.nii.gz'),
+                                      '-ofolder', str(tmp_path),
+                                      '-qc', str(tmp_path)])
+    assert pytest_wrapped_e.type == SystemExit
```

Comment:
I think this assertion doesn't really do anything, because we know that the type will always be `SystemExit` if we're able to reach the assertion.

(If the error's type _isn't_ `SystemExit`, then the `pytest.raises()` call will instead throw an error, and so the assertion will never be reached.)

Reply:
Typically, for `parser.error`, I think we usually do this instead:

```suggestion
    assert pytest_wrapped_e.value.code == 2
```

This still doesn't do a lot, but it at least ensures that the `SystemExit` is likely coming from `argparse` (which uses error code 2).

Reply:
Oh wait, sorry, nevermind. I just realized you're using `printv` instead of `parser.error`. One sec, let me add a follow-up comment.